lighthouse-core/gather/drivers/extension.js

```diff
@@ -130,19 +130,23 @@ class ExtensionDriver extends Driver {
   }
 
   queryCurrentTab_() {
-    const currentTab = {
-      active: true,
-      windowId: chrome.windows.WINDOW_ID_CURRENT
-    };
-
     return new Promise((resolve, reject) => {
-      chrome.tabs.query(currentTab, tabs => {
+      const queryOpts = {
+        active: true,
+        lastFocusedWindow: true,
+        windowType: 'normal'
+      };
+
+      chrome.tabs.query(queryOpts, (tabs => {
         if (chrome.runtime.lastError) {
           return reject(chrome.runtime.lastError);
         }
-
+        if (tabs.length === 0) {
+          const message = 'Couldn\'t resolve current tab. Please file a bug.';
```

Comment:
how often does this happen in practice? It worries me you added this as an explicit error case where I haven't seen this problem in a while on master :)